Userland/which.cpp

```diff
@@ -0,0 +1,32 @@
+#include <AK/AKString.h>
+#include <AK/ScopedValueRollback.h>
+#include <AK/Vector.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+int main(int argc, char** argv)
+{
+    if (argc < 2) {
+        printf("usage: which [executable]\n");
```

Comment:
It's convention for mandatory arguments to be written `<like this>`.
Optional arguments are written `[like this]`.